src/collections/_documentation/performance-monitoring/configuration/node.md

```diff
@@ -26,10 +26,26 @@ const Apm = require("@sentry/apm");
 
 Sentry.init({
   dsn: "___PUBLIC_DSN___",
-  tracesSampleRate: 0.25
+  tracesSampleRate: 1.0
 });
+
+// Your test code to verify it works
+
+const transaction = Sentry.startTransaction({
+  op: 'test',
+  name: 'My First Test Transaction',
+});
+
+setTimeout(() => {
+  transaction.finish();
```

Comment:
I'm not sure about the order here, but if `foo` fails, then transaction will never finish. Could be wrapped in try/catch, but it's just a nitpick.

Reply:
The transaction actually does get finished.  Both the transaction and error event do get generated and submitted to Sentry.

The error event won't get tracing information though.

Reply:
In this order yes, but if we'd change that `foo()` is called first, and it throws, then transaction will never be finished.